src/MassTransit.AmazonSqsTransport/Configuration/DelayedMessageSchedulerExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright 2007-2016 Chris Patterson, Dru Sellers, Travis Smith, et. al.
+//  
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the 
+// License at 
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0 
+// 
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR 
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the 
+// specific language governing permissions and limitations under the License.
+namespace MassTransit.AmazonSqsTransport.Configuration
+{
+    using System;
+    using Specifications;
+
+
+    public static class DelayedMessageSchedulerExtensions
+    {
+        /// <summary>
+        /// Uses the AmazonSqs' delayed messages to schedule messages for future delivery. A lightweight
+        /// alternative to Quartz, which does not require any storage outside of AmazonSqs.
+        /// </summary>
+        /// <param name="configurator"></param>
+        public static void UseDelayedMessageScheduler(this IBusFactoryConfigurator configurator)

```

Comment:
Could we rename the extension method to something like `UseAmazonSqsMessageScheduler` like the [UseServiceBusMessageScheduler](http://masstransit-project.com/MassTransit/usage/scheduling/azure-sb-scheduler.html#configuring-the-enqueue-time-scheduler) or `UseDelayedSqsMessageScheduler` like [UseDelayedExchangeMessageScheduler](http://masstransit-project.com/MassTransit/usage/scheduling/rabbitmq-delayed.html#scheduling-messages) so it clear what stack it is using to delay the messages

Reply:
Agree. I've renamed it into `UseAmazonSqsMessageScheduler`.